packages/twenty-front/src/modules/settings/components/SettingsOptions/__stories__/SettingsOptionCardContentToggle.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import { SettingsOptionCardContentToggle } from '@/settings/components/SettingsOptions/SettingsOptionCardContentToggle';
```

Comment:
Did you bring in files from a different PR? thanks

Reply:
https://github.com/twentyhq/twenty/pull/8456
Yes! My last PR was merged, but no stories! so I added them here.